paypal/standard/pdt/forms.py

```diff
@@ -12,3 +14,9 @@ class Meta:
         exclude = ['ipaddress', 'flag', 'flag_code',
                    'flag_info', 'query', 'response',
                    'created_at', 'updated', 'form_view']
+
+
+class PayPalPDTCallbackForm(forms.ModelForm):
+    class Meta:
+        model = PayPalPDT
+        fields = ['amt', 'cm', 'tx', 'st']
```

Comment:
If I've read it correctly, the only piece of information that we use is `tx` (looking at `PayPalPDT._postback()`. Do you agree, if so, should we make that the only field? Are all the other fields set from the postback call?

(Please forgive my ignorance, I've only ever personally used IPN)

It would be great, either way, if we could have a reason for the specific set of fields included here, and a comment on `PayPalPDTCallbackForm` that explained its purpose.

Reply:
I can't remember my logic, but I assume I chose these four to match the PDT-specific fields here:

https://github.com/spookylukey/django-paypal/blob/9c75afe07685dd51c8be11ba1e6a92aec068a0d8/paypal/standard/pdt/models.py#L33-L37

I suspect I omitted `sig` because it doesn't appear in either the old-style or new-style query parameters.

As you say, `tx` is the only field used explicitly, but the others are saved to the instance, and I wanted to avoid introducing any changes in behaviour in this PR.

Having said all that, I'm not convinced that `PayPalPDT` needs these custom fields at all — all the interesting information comes from the postback and is saved to fields defined in `PayPalStandardBase`.